drivers/fuel_gauge/sbs_gauge/sbs_gauge.c

```diff
@@ -179,7 +179,7 @@ static int sbs_gauge_get_prop(const struct device *dev, struct fuel_gauge_get_pr
 
 static int sbs_gauge_do_battery_cutoff(const struct device *dev)
 {
-	int rc;
```

Comment:
Actually can we change this to be `-ENOSUP` or some other error?

Reply:
Changed it to `-ENOTSUP`.